python/sglang/srt/managers/schedule_batch.py

```diff
@@ -1311,8 +1622,13 @@ def prepare_for_extend(self):
                 continue
             for mm_item in mm_input.mm_items:
                 pixel_values = getattr(mm_item, "feature", None)
+                # logger.info("pixel_value.device :{} vs self.device {}".format(pixel_values.device, torch.cuda.current_device()))
                 if isinstance(pixel_values, torch.Tensor):
                     mm_item.feature = pixel_values.to(self.device, non_blocking=True)
+                elif isinstance(pixel_values, CudaIpcTensorTransportProxy):
```

Comment:
I think here CudaIpcTensorTransportProxy should be a type of CudaIpcTransportProxyTensor.
But since we already have TransportProxyTensor, CudaIpc is one type, another is default. Why we don't reuse it?
Just a question mark, don't need to do refactor, since we've already gone so far.